functests/sctp/sctp.go

```diff
@@ -73,6 +79,11 @@ var _ = Describe("sctp", func() {
 
 		err = namespaces.Clean(testNamespace, client.Client)
 		Expect(err).ToNot(HaveOccurred())
+
+		newSCTPNodeSelector, err := findSCTPNodeSelector()
```

Comment:
Let's be more consistent here.
If the env variable is set, it must have priority because the user willingly set it. Otherwise we try to do the discovery.